### PR TITLE
Mark broadcast notifications in the list

### PR DESCRIPTION
**Pull request by mareklibra**
## Hey, I just made a Pull Request!

This patch adds a little icon to mark a broadcast notifications in the list.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by mareklibra** [comment]
Screenshot from 2024-08-14 14-52-19

File: .changeset/healthy-islands-smash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications': minor
+---
+
+Broadcast notifications are newly decorated with an icon.
```
Comment (Rugvip) [inline .changeset/healthy-islands-smash.md:5]:
```suggestion
---
'@backstage/plugin-notifications': patch
---

Broadcast notifications are now decorated with an icon.
```

**Review by Rugvip — APPROVED** [review]
🎉

File: plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx [hunk 2]
```diff
@@ -237,6 +242,12 @@ export const NotificationsTable = ({
                     </Typography>
                   ) : null}
                   <Typography variant="caption">
+                    {!notification.user && (
+                      <>
+                        <BroadcastIcon className={classes.broadcastIcon} />
+                        &nbsp;
```
Comment (freben) [inline plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx:248]: Maybe consider a marginRight on the style instead

File: plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
 import { Notification } from '@backstage/plugin-notifications-common';
 import { useConfirm } from 'material-ui-confirm';
+import BroadcastIcon from '@material-ui/icons/Cast'; /* TODO: Change to Podcasts or CellTower once updated */
```
Comment (stephenglass) [inline plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx:27]: Have you considered the `RssFeedIcon`?
image

File: .changeset/healthy-islands-smash.md (hunk 1 above)
Comment (mareklibra) [reply .changeset/healthy-islands-smash.md]: done

File: plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx (hunk 2 above)
Comment (mareklibra) [reply plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx]: done

File: plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx (hunk 3 above)
Comment (mareklibra) [reply plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx]: Thanks, looks better.

**Review by freben — APPROVED** [review]

**Pull request merged**